apps/mkt/components/automation-carousel/automation-icon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { TablerIconsProps } from "@tabler/icons-react";
+
+interface AutomationIconProps extends TablerIconsProps {
+  IconComponent: React.FC<TablerIconsProps>;
```

Comment:
Instead of receiving an icon component as a property, we want a string to indicate what kind of benefit the icon is about.

Take a look at https://github.com/sweetr-dev/sweetr.dev/blob/main/apps/web/src/app/automations/components/icon-automation-benefit/icon-automation-benefit.tsx

Would be something like this:
```suggestion
  type: 'cycleTime' | 'failureRate' | 'compliance' | 'security' | 'techDebt';
```

then you'll need additional logic to display the correct icon. See the getIcon method in https://github.com/sweetr-dev/sweetr.dev/blob/main/apps/web/src/app/automations/components/icon-automation-benefit/icon-automation-benefit.tsx#L24